Utilities/DataLoading/EdFi.LoadTools/BulkLoadClient/LoadProcess.cs

```diff
@@ -173,24 +183,23 @@ private static IEnumerable<string> GetFilesToImport(BulkLoadClientConfiguration
 
                 xmlReaderSettings.ValidationEventHandler += (s, e) => { validationErrors.Add(e.Message); };
 
-                using (var fileStream = new FileStream(fileToImport, FileMode.Open, FileAccess.Read))
+                await using var fileStream = new FileStream(fileToImport, FileMode.Open, FileAccess.Read);
+
+                using (var reader = XmlReader.Create(fileStream, xmlReaderSettings))
                 {
-                    using (var reader = XmlReader.Create(fileStream, xmlReaderSettings))
+                    while (reader.Read())
                     {
-                        while (reader.Read())
+                        if (reader.NodeType == XmlNodeType.Element && unsupportedInterchanges.Any(
+                            i => i.Equals(reader.Name, StringComparison.InvariantCultureIgnoreCase)))
                         {
-                            if (reader.NodeType == XmlNodeType.Element && unsupportedInterchanges.Any(
-                                i => i.Equals(reader.Name, StringComparison.InvariantCultureIgnoreCase)))
-                            {
-                                validationErrors.Add($"Import of data for {reader.Name} is not supported");
-                            }
+                            validationErrors.Add($"Import of data for {reader.Name} is not supported");
                         }
-
-                        reader.Close();
                     }
 
-                    fileStream.Close();
+                    reader.Close();
                 }
+
+                fileStream.Close();
```

Comment:
so the recommendation is not call close, and let the dispose do the work.
```
Closes the current stream and releases any resources (such as sockets and file handles) associated with the current stream. Instead of calling this method, ensure that the stream is properly disposed.(Inherited from Stream)
```
I see value in calling `reader.FlushAsync()` instead.